### PR TITLE
Fix shrinking SDG images on About Page

### LINKED ISSUE #861: About Us -> Fix shrinking images in SDG section
### Overview
In SDG section, left row of images gets smaller when minimizing the page before it switches over to mobile view.

Screen Shot 2020-12-10 at 1 03 16 PM

### Action Items
Make it stop doing that.

### Resources/Instructions
https://www.hackforla.org/about-us

### PR DESCRIPTION
**Pull request by averdin2**
Fixes #861 

- Fixed shrinking images in SDG section on the about page. 
- Also fixed some styling with numbers and text in the SDG section (See: https://github.com/hackforla/website/issues/861#issuecomment-822847982).
- Moved LA Mayor link to be bottom centered with SDG images (See: https://github.com/hackforla/website/issues/861#issuecomment-822855821).

### CODE REVIEW

**Comment by averdin2** [comment]
Going to resolve merge conflicts now that #1424 has been merged, which was blocking my progress.

**Comment by averdin2** [comment]
Merged changes made by PR #1424 and ready for review.

File: _sass/components/_about.scss [hunk 1]
```diff
@@ -767,14 +774,6 @@ a.anchor {
     flex-direction: row;
   }
 
-  .sdg-left {
```
Comment (averdin2) [inline _sass/components/_about.scss:770]: I want to add a note for the reviewer here. I don't know why it looks like this sdg-left and sdg-right was removed. I changed the values of sdg-left to width: 15% and sdg-right to width 85% to allow spacing for numbers and text in the SDG section.

**Review by erikaBell — APPROVED** [review]
Looks good! Only Comment @averdin2,  some before & after photos (-;

Great Job!

**Pull request merged**